functions/src/submitApplication/models.js

```diff
@@ -13,7 +13,7 @@ exports.Hacker = function Hacker(data) {
       city: data.city,
       school: data.school,
       gender: data.gender,
-      birthDate: data.birthDate,
+      birthDate: data.birthday,
```

Comment:
Can we rename `birthDate` to `birthdate`? As well as replace every `birthday` in the project (in web and functions) with `birthdate`? I intended to do that when I initially created `model.js` but I messed up by not updating `birthday` in the front-end, and by capitalizing the D in `birthdate` 😕

Reply:
I think that's getting out of the scope of this PR (because this PR is just focused on changing the way we store the hacker application in firestore) - we could make a separate issue for it. I personally think that's a low priority task bc it doesn't add any value/functionality for end users, and especially given that we have a very tight deadline. What do you think?

Reply:
I think we should fix at least the capitalization of `birthDate` so we don't have `birthdate` and `birthDate` in our database if we end up doing the task.

Reply:
Why would we have both `birthdate` and `birthDate` in our database? I'm not changing the name of the field in firestore, it's still `birthDate` like before

Reply:
Yeah I'm saying that we should rename it from `birthDate` to `birthdate` lol

Reply:
Oh.. but why do you want to change it? I’m just reluctant bc we would have to backfill. If we change it from birthDate to birthdate then all the applications before this PR have the birthDate field while all the applications after this PR will have the birthdate field instead?

Reply:
The `birthDate` field wasn't written to the DB at all because of the typo right? Previously it was       `birthDate: data.birthDate` but `data.birthDate` was actually `data.birthday` on the front-end, so the field ended up being null and removed by deep-cleaner before the database write.

Reply:
So I see this as a chance to fix the grammar before we start writing the field, super trivial but yeah

Reply:
Ohhhhh lol yeah I totally forgot, forget what I said about backfilling